.github/workflows/pipeline.yml

```diff
@@ -41,8 +41,7 @@ jobs:
       # Check linting and formatting
       - name: Check linting and formatting
         run: |
-          black ./ --check
-          pylint lexi_meter/
+          source run_style.sh
```

Comment:
Do we want code to be changed during a PR? Or rather have the build fail if it is not formatted?